src/dynamo0p3.py

```diff
@@ -3436,8 +3592,9 @@ def iteration_space_arg(self, mapping=None):
 
         # Since we always compute operators out to the L1 halo we first
         # check whether this kernel writes to an operator
-        op_args = self.args_filter(arg_types=["gh_operator"],
-                                   arg_accesses=["gh_write", "gh_inc"])
+        op_args = psyGen.args_filter(self._args,
+                                     arg_types=["gh_operator"],
```

Comment:
I'd only touched this line because of changing ``args_filter()`` to be a module function rather than a method. However, you're right so I've now changed the ``arg_types`` argument too.